src/interfaces/slack_bot/.env.example

```diff
@@ -0,0 +1,23 @@
+# Environment variables declared in this file are automatically made available to Prisma, and to the application when NODE_ENV=development.
+# See README.md and https://pris.ly/d/prisma-schema#accessing-environment-variables-from-the-schema
+
+TOOLKIT_API_HOST=http://0.0.0.0:8000
+COHERE_ORGANIZATION_ID=<cohere-organization-id>
+COHERE_SLACK_TEAM_ID=<cohere-slack-team-id>
+SLACK_SIGNING_SECRET=<signing-secret>
+FEEDBACK_CLIENT_TAG=<feedback-client-tag>
+COMMAND_SUFFIX=<app-name>
+SENTRY_DSN=<sentry-dsn>
+DATABASE_URL="postgresql://coralslack:password@localhost:5432/coralslackdb"
```

Comment:
Can we change the wording from coral?

Reply:
It has been changed to not have "coral" in `DATABASE_URL`.